### PR TITLE
Don't allow multiple plugin configs

### PR DESCRIPTION
**Pull request by timgl**
## Changes

[A user reported having some plugins run twice](https://posthogusers.slack.com/archives/C01GLBKHKQT/p1643140118334000), which I think is due to the frontend POSTing duplicate plugin configs

- If the frontend tries POSTing a plugin config for a plugin that already has one, just return the existing one instead
- Migration to remove all duplicate plugin configs + test

@yakkomajuri I think running the migration should solve this issue, can you check it's safe?

## How did you test this code?

Unit tests

### CODE REVIEW

**Comment by yakkomajuri** [comment]
Looking at this but my first comment would be could we prioritize keeping configs that have `enabled = true`?

**Comment by timgl** [comment]
@yakkomajuri Good catch, done!

File: posthog/migrations/0204_remove_duplicate_plugin_configs.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+from django.db import migrations, models
+
+from posthog.constants import AnalyticsDBMS
+
+
+def remove_duplicate_plugin_configs(apps, schema_editor):
+    PluginConfig = apps.get_model("posthog", "PluginConfig")
+    configs = PluginConfig.objects.raw(
+        """
+    select * from posthog_pluginconfig ou
```
Comment (yakkomajuri) [inline posthog/migrations/0204_remove_duplicate_plugin_configs.py:13]: tiny nit that I'd format this statement better in order to make it nicer to read, like adding indentation for the subquery

File: posthog/test/test_migration_0204.py [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+from django.apps import apps
+from django.db import connection
+from django.db.migrations.executor import MigrationExecutor
+from django.test import TestCase
+
+from posthog.test.base import BaseTest
+
+"""
+If you're looking at this, you can probably remove this
```
Comment (yakkomajuri) [inline posthog/test/test_migration_0204.py:11]: :D

**Review by yakkomajuri — APPROVED** [review]
migration looks good to me, thanks for spotting and fixing this

**Pull request merged**